packages/main/src/types/TabsOverflowMode.js

```diff
@@ -0,0 +1,40 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.main.types.TabsOverflowMode.prototype
+ * @public
+ */
+const TabsOverflowModes = {
+	/**
+	 * End type if there is only one overflow in the end of the scroll container .
```

Comment:
Please rephrase this sentence completely. Also, do not speak of "scroll container" as this is an implementation detail apps will be unaware of.